### PR TITLE
release script and releasing.md instructions

### PR DESCRIPTION
**Pull request by aabmass**

### CODE REVIEW

File: docs/releasing.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+**pointing at the first commit (a)** that was merged into master. For the
+example PR listed above, that creates release
+[`v0.11b0@4ad9ccd`](https://github.com/GoogleCloudPlatform/opentelemetry-operations-python/releases/tag/v0.11b0).
+- In description, paste a changelog for the packages. I used this (probably
+buggy) small script for the example PR's tag:
+
+  ```bash
+  for cl in opentelemetry-*/CHANGELOG.md; do
+      echo "# `dirname $cl`"
+      pcregrep "## Version 0\.11b0[\s\S]+?\K^-[\s\S]+?(?=(##|\Z))" $cl
```
Comment (aabmass) [inline docs/releasing.md:92]:
```suggestion
      pcregrep -M "## Version 0\.11b0[\s\S]+?\K^-[\s\S]+?(?=(##|\Z))" $cl
```

Comment (aabmass) [inline docs/releasing.md:92]:
```suggestion
      if cl_entries=`pcregrep -M -o1 "^## Version 0\.10b0$\n\n^Released.*\n\n((?:- [\s\S]+?)*?)(?=(\s+##|\Z))" $cl`
      then
          echo -e "# `dirname $cl`\n$cl_entries"
      fi
```

File: docs/releasing.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+commit.
+
+```bash
+# pull in the new release tag
+git fetch origin
+
+# create branch and move stable
+git branch release/0.11b0 v0.11b0
+git tag -d stable
+git tag stable v0.11b0
```
Comment (aabmass) [inline docs/releasing.md:109]:
```suggestion
git push origin :refs/tags/stable
git tag stable v0.11b0
git push --tags
```

File: docs/releasing.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    pushd `dirname $setup_file`
+    # to be safe
+    rm -rf dist/ build/
+    python setup.py sdist bdist_wheel
+    popd
+done
+
+# See what was built
+ls opentelemetry-*/dist/*
+
```
Comment (aabmass) [inline docs/releasing.md:142]: add `twine check`

File: docs/releasing.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+git push --set-upstream origin release/0.11b0 stable
+```
+
+## Push to PyPI
+
+TODO: incorporate these steps into `releasing.py`.
+
+Finally, publish the packages. Use PyPI user
+[`google_opentelemetry`](https://pypi.org/user/google_opentelemetry/),
+consulting internal docs for how to get a login token.
```
Comment (aabmass) [inline docs/releasing.md:123]: how to get login info for `TWINE_USERNAME`, `TWINE_PASSWORD`.

Comment (aabmass) [reply docs/releasing.md:123]: make sure you're checkout out to the release tag

**Comment by aabmass** [comment]
@kintel, did you still want to take a look?

**Comment by kintel** [comment]
This is not a domain I'm familiar with. If you feel confident as it, feel free to merge it. If you think I should give it a checkmark first, let me know.

**Pull request merged**